editor/app/(dev)/ui/floating-window/page.tsx

```
@@ -0,0 +1,172 @@
+                          >
+                            <X className="h-4 w-4" aria-hidden />
+                            <span className="sr-only">Close</span>
+                          </FloatingWindowClose>
+                        </FloatingWindowTitleBar>
+                        <FloatingWindowBody className="font-mono text-xs space-y-2">
+                          <p className="text-muted-foreground">
+                            Drag me independently; no shared context required.
+                          </p>
+                          <div className="rounded-md bg-slate-950 text-slate-100 p-3">
+                            <div>$ window.moveTo(x, y)</div>
+                            <div className="text-slate-400">// logs</div>
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)